internal/controllers/machine/machine_controller.go

```diff
@@ -55,8 +55,8 @@ import (
 )
 
 const (
-	// MachineControllerName defines the controller used when creating clients.
-	MachineControllerName = "machine-controller"
+	// ControllerName defines the controller used when creating clients.
+	ControllerName = "machine-controller"
```

Comment:
```suggestion
	controllerName = "machine-controller"
```

I think this doesn't have to be exported anymore (no external usages in our code & it's in an internal package now)

Reply:
fixed